style/app.scss

```diff
@@ -642,186 +642,168 @@ a:active
-  height           : 100%;
-  float            : left;
-  background-color : #7a6a96;
-  position         : absolute;
-  padding-left     : 23px;
+  height : 0px;
+  width  : 0px;
 }
 
-#sidebar-button
+.collapsed > #sidebar > #reset-selections
 {
-  cursor           : pointer;
-  display          : inline-block;
-  width            : 40px;
-  height           : 100%;
-  float            : left;
-  background-color : #5c497e;
-  position         : absolute;
-  border           : solid 1px #000000;
+  height : 0px;
+  width  : 0px;
+  margin : 0px;
 }
 
-#sidebar-button:hover
+.collapsed > #sidebar > #sidebar-button
 {
-  background-color : #cac4d4;
+  margin-top : -25px;
 }
 
-#sidebar-icon
+.expanded > #sidebar > #sidebar-button
 {
-  width        : 30px;
-  height       : 50px;
-  padding-top  : 20px;
-  padding-left : 2px;
-  position     : absolute;
-  top          : 40%;
-  left         : 4px;
+  margin-top : -5px;
 }
 
-#focuses-nav, #graph-nav
-{
-  cursor : pointer;
-}
-
-#sidebar-nav
+#sidebar-button
 {
-  width            : 100%;
-  font-size        : 13px;
-  background-color : #cac4d4;
-  border           : solid 1px #dedede;
-  box-shadow       : 0 2px 2px -1px rgba(0, 0, 0, 0.055);
-  display          : block;
-  overflow         : hidden;
+  cursor           : pointer;
+  border-radius    : 10px;
+  width            : 30px;
+  height           : 15px;
+  background-color : #d9d9d9;
+  margin-left      : 60px;
+  text-align       : center;
+  visibility       : visible !important;
 }
 
-#sidebar-nav ul
+#selected-courses
 {
-  width  : 100%;
-  margin : 0 0 0 0;
+  margin-top       : 10px;
+  margin-bottom    : 10px;
+  font-size        : small;
+  color            : #a47cb8;
 }
 
-#sidebar-nav ul li
+.courses
 {
-  list-style-type    : none;
-  display            : inline-block;
-  width              : 48%;
-  -webkit-transition : all 0.2s;
-  -moz-transition    : all 0.2s;
-  -ms-transition     : all 0.2s;
-  -o-transition      : all 0.2s;
-  transition         : all 0.2s;
+  overflow         : auto;
+  height           : 275px;
+  width            : 105px;
+  display          : inline-block;
 }
 
-#sidebar-nav ul li:hover
+#course-selection
 {
-  background-color : #5C497E !important;
+  width: 105px;
+  font-size: small;
+  text-align: center;
 }
 
-#sidebar-nav ul li:hover a
+#sidebar-button:hover
 {
-  color : white !important;
+  background-color: #cac4d4;
 }
 
-#sidebar-nav ul li .nav-section
+#sidebar-icon
 {
-  color           : #000000;
-  display         : inline-block;
-  line-height     : 30px;
-  text-align      : center;
-  width           : 95%;
-  text-decoration : none;
+  width : 15px;
+  height: 15px;
 }
 
-#focuses, #graphs
+#fcecount
 {
-  margin-top  : 25px;
-  margin-left : 25px;
-  height      : 86%;
-  overflow-y  : scroll;
+  border-radius    : 10px;
+  line-height      : 40px;
+  height           : 40px;
+  width            : 140px;
+  float            : left;
+  background-color : #ffffff;
```

Comment:
Once again, you can delete this rule once you've implemented the other changes I've mentioned.

Reply:
You missed this comment

Reply:
My apologies, I did not elaborate further. With respect to this comment, I tried removing it but this messed up the centring of the text vertically within the div. This is why I set line height to equal the height of the <div> itself as well as the relation between FCE and button because even while collapsed sidebar is a flex box. Should I rework the collapsed side of things to remove the need for line-height?
<img width="173" alt="Screen Shot 2021-07-26 at 10 22 27 AM" src="https://user-images.githubusercontent.com/32244859/127004876-e6cc478f-2316-4229-ba37-c122f21e30f0.png">